cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f08ac68",
+   "metadata": {},
+   "source": [
+    "# Itteratively Find Improved Consensus Labels for Multiannotator Data using Cleanlab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44a0555b",
+   "metadata": {},
+   "source": [
+    "This example shows how to improve consensus labels by combining the CROWDLAB algorithm with itteratively retraining a model. For an introductory tutorial on finding consensus labels with the multiannotator library see [Find Best Consensus Labels for Multiannotator Data using Cleanlab](linke to hui wen's). \n",
+    "\n",
+    "The following code uses the [cifar10h](linke cifar10h) multiannotator labeling dataset which is a collection of 2751 annotators each labeling 200 examples for all 10,000 test images of the original [cifar10](link cifar10) dataset but **any multiannotator classification image dataset should work with the code below**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95724ad1",
+   "metadata": {},
+   "source": [
+    "## 1. Install and import required dependencies, build example folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68b291ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from matplotlib import pyplot as plt\n",
+    "from datetime import datetime\n",
+    "\n",
+    "from utils.model_training import train_model\n",
+    "from utils.model_training import sum_xval_folds\n",
```

Comment:
[nit] these can be combined into one line:
`from utils.model_training import train_model, sum_xval_folds`

Reply:
Sounds great is that the Cleanlab standard on this? 

There's also `from utils import model_training` with `model_training.train_model()`

Reply:
ya that's generally preferred to do all the imports at once from the same file